app/views/sub_guide_cards/show.html.erb

```diff
@@ -1,3 +1,7 @@
 <h1>SubGuideCards#show</h1>
 <p>Find me in app/views/sub_guide_cards/show.html.erb</p>
 <%= @sub_guide_card.heading %>
+<%= @sub_guide_card.path %>
```

Comment:
Do we want to print the sub guide card path on the show page? Maybe it was printed for debugging and we can remove it?